docs/Principles/modeling-principles.md

```diff
@@ -0,0 +1,27 @@
+# Modeling Principles
+
+## Conforms to Schema
+The Monarch Biolink Specification is an implementation of the Biolink Model.
+The KG must be conformant with The Monarch Biolink Specification.
+
+## Node Normalization
+The final KG must have Nodes normalized to the canonical prefix for any given node type.
+The canonical prefix should be determined by The Monarch Biolink Model Specification.
+
+## Authoratative Source
+Providers of Associations are not the authoratative sources for the Nodes in general. Nodes should be ingested from their
```

Comment:
authoritatative -> authoritative (several instances)

Same comment about Associations: if this is used as a normal English word, it should have a lowercase a. If it means something special in this context, then it should be defined.